xml2kinto/__main__.py

```diff
@@ -12,14 +14,27 @@
 fields = ('subject', 'publicKeyHash', 'serialNumber', 'issuerName')
 
 
-def main():
+def main(args=None):
+    parser = argparse.ArgumentParser(description='Syncs a Kinto DB.')
+
+    parser.add_argument('-s', '--kinto-server', help='Kinto Server',
+                        type=str, default=kinto_server)
+
+    parser.add_argument('-x', '--xml-file', help='XML Source file',
+                        type=str, default=xml_file)
+
+    parser.add_argument('-a', '--auth', help='BasicAuth user:pass',
```

Comment:
We should probably access both ``--auth-type`` and ``--auth`` like requests does.
Then if we want to use FxA in production we would be able to.